script.js

```diff
@@ -0,0 +1,49 @@
+var clickMe = document.querySelector('#first_paragraph');
+clickMe.addEventListener('click', function (){
+  console.log('h1 got clicked !!!!');
+  document.body.style.background = "blue";
+})
+
+var clickMe = document.querySelector('second_paragraph');
+clickMe.addEventListener('click', function (){
+  console.log('h2 got clicked!!!!');
+  document.body.style.background = "pink";
+})
```

Comment:
Notice in that "#second_paragraph" (ttmar_starter.html line #25) has `style="display: none"`
Instead of changing we the background color of the page we could change this display property when we click on the link. This also assumes using better variable names instead of them all being `clickMe`
```js
var secondParagraph = document.querySelector('second_paragraph');
clickMe.addEventListener('click', function (){
  secondParagraph.style.display = "block";
})
```